### PR TITLE
An example comparison for docs

### PR DESCRIPTION
**Pull request by dphov**

### CODE REVIEW

**Review by Timmyfbaby12 — COMMENTED** [review]
Reviewers

**Comment by Timmyfbaby12** [comment]
image

**Comment by khaiminn1589** [comment]
QTu12SdX7VM

File: lib/linguist/heuristics.rb [hunk 1]
```diff
@@ -23,6 +23,15 @@ def self.call(blob, languages)
       [] # No heuristics matched
     end
 
+    # Public: Do we have a heuristic for these candidate languages?
+    #
+    # languages - Array of Language objects
+    #
+    # Returns true if there are matching heuristics or nil
+    def self.defined_for?(languages)
+      @heuristics.select { |h| h.matches?(languages) }.any?
```
Comment (Shalini9623) [inline lib/linguist/heuristics.rb:32]: Add space -shalini

**Comment by Grochu616** [comment]
> description

**Comment by dilan2468101357-jpg** [comment]
Iwjsujseowiwnavs

**Comment by dilan2468101357-jpg** [comment]
Planetario de amor

**Pull request closed without merging**